backend/app/controllers/MenteesController.php

```diff
@@ -75,4 +176,50 @@ public function addMentee()
             ]);
         }
     }
+
+    public function firstLogin()
+    {
+
+        $users_id = auth()->id();
+        $initial_weight = request()->get('initial_weight');
+        $current_weight = request()->get('current_weight');
+        $target_weight = request()->get('target_weight');
+        $growth = request()->get('growth');
+        $age = request()->get('age');
+        $activity_factor = request()->get('activity_factor');
+        $sex = request()->get('sex');
```

Comment:
method `get()` accepts an array.
```php
request()->get(["initial_weight", "current_weight", "target_weight", "growth", "age", "activity_factor", "sex"]);
```

Reply:
I know, but what's the difference? Since I immediately assign it to variables.

Reply:
This is against the DRY rule :)

Reply:
Okay, but after adding validation from leaf those lines disappear.